src/components/screens/HistoryScreen/mappers/changeTrust.tsx

```diff
@@ -54,36 +45,17 @@ export const mapChangeTrustHistoryItem = ({
     />
   );
 
-  const actionText = isRemovingTrustline
-    ? t("history.transactionHistory.removedTrustline")
-    : t("history.transactionHistory.addedTrustline");
+  const ActionIcon =
+    Icon[historyItemData.actionIconString as keyof typeof Icon];
 
   const ActionIconComponent = isRemovingTrustline ? (
```

Comment:
this ternary isn't necessary anymore - but I left it in to show that if you needed to do some additional configuration, it's still quite easy to do